phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -28,9 +30,12 @@
     @Value("${application.name}")
     private String applicationName;
 
+    @Autowired
```

Comment:
ViewsConfiguration and ViewsLibraryConfiguration will depend on this new configuration